### PR TITLE
chore: propose detail job navigation

### PR DESCRIPTION
**Pull request by maulanasdqn**
I propose way to navigate in detail job

i propose using section in detail job as params

so we can detect the active tab by params such as `http://localhost:3000/clvu7pptw0001whkpfzytmbl4/job/clvv3tof00000ai7m0vnzbhb3/all-applicant`

it's mean the all-aplicant tab is active because the params tab all-aplicant was selected

more detail in code

**TASK**

- [x] Slicing Send Email

Screenshot
image
image

image
image
image
image
image

### CODE REVIEW

File: app/(dashboard)/[orgId]/(routes)/job/[id]/[tab]/layout.tsx
Comment (maulanamaleek) [inline app/(dashboard)/[orgId]/(routes)/job/[id]/[tab]/layout.tsx:1]: mas ini layout nya masih muncul kalau route nya ga terdaftar. mungkin bisa diarahin ke `404` page, atau nanya ke Designer untuk desain `404` di halaman job detail

image

Comment (maulanasdqn) [reply app/(dashboard)/[orgId]/(routes)/job/[id]/[tab]/layout.tsx:1]: nah iya bisa mas arahain ke 404

Comment (maulanasdqn) [reply app/(dashboard)/[orgId]/(routes)/job/[id]/[tab]/layout.tsx:1]: ahh iya okee mas

**Comment by maulanamaleek** [comment]
LGTM 👍

**Pull request merged**